kyco/config.py

```diff
@@ -16,6 +17,8 @@
 else:
     BASE_ENV = '/'
 
+# include napps modules
+sys.path.append(os.path.join(BASE_ENV, 'var/lib/kytos'))
```

Comment:
`os.path.join(BASE_ENV, 'var/lib/kytos')` is already been used on many parts of code. Please make this a var or config.